corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yaml

```diff
@@ -149,12 +149,25 @@
   since: '2.14'
 
 - id: use_grid_menus
-  name: Use Menu Grids
-  description: Display root menu as a grid instead of a list.
-  widget: bool
+  name: Modules Menu Display
+  description: Display root menu as a list or grid.
+  values: [false, true]
+  value_names: ['List', 'Grid']
   default: false
   since: "2.24"
   toggle: GRID_MENUS
+  foot_note: "Set Grid Menus as No for this to be active"
```

Comment:
On second thought, for the `Set Grid Menus as No for this to be active` text specifically, you can remove it and instead use the `requires` property. If you look under "Java Phone General Settings" you'll see that "Auto-Sync Frequency" and "Unsynced Time Limit" are disabled if "Sync Mode" is set to "Push Only." This leads to a display like this:
<img width="691" alt="screen shot 2016-09-20 at 5 33 37 pm" src="https://cloud.githubusercontent.com/assets/1486591/18689788/f530fa06-7f58-11e6-826d-387619926a5b.png">

The `For per-module option...` text for the setting below is still needed, but can be moved into that setting's description.